apps/cyberstorm-storybook/stories/components/Switch.stories.tsx

```diff
@@ -1,42 +1,27 @@
 import { StoryFn, Meta } from "@storybook/react";
 import { Switch } from "@thunderstore/cyberstorm";
-import React, { CSSProperties, useState } from "react";
+import React, { useState } from "react";
 
 export default {
   title: "Cyberstorm/Components/Switch",
   component: Switch,
 } as Meta<typeof Switch>;
 
-const backgroundStyle: CSSProperties = {
```

Comment:
Ideally the cleanup would've been done in a separate commit to make reviewing the "meat" of the PR easier. This is just for future reference, doesn't require action now.